RAGchain/benchmark/dataset/dstc11.py

```diff
@@ -0,0 +1,176 @@
+import itertools
+from copy import deepcopy
+from typing import List, Optional
+
+from datasets import load_dataset
+
+from RAGchain.DB.base import BaseDB
+from RAGchain.benchmark.dataset.base import BaseDatasetEvaluator
+from RAGchain.pipeline.base import BaseRunPipeline
+from RAGchain.retrieval.base import BaseRetrieval
+from RAGchain.schema import EvaluateResult, Passage
+
+
+class DSTC11Evaluator(BaseDatasetEvaluator):
```

Comment:
You must mention Track5 for this evaluator, not only DSTC11
There are total 5 tracks in DSTC11